build/relayers/ibc-relayers.md

```diff
@@ -15,6 +15,7 @@ We encourage everyone to follow and support our IBC-relayer operators
 | [CryptoCrew Validators](http://ccvalidators.com/)         | <p><a href="mailto:cs@ccvalidators.com">cs@ccvalidators.com</a><br><img src="https://user-images.githubusercontent.com/95667791/163506878-e7e278b2-1cf6-4dac-bc13-e92f63600ee0.png" alt="Twitter" data-size="line"> Twitter <a href="http://twitter.com/crypto_crew">@crypto_crew</a><br><img src="https://user-images.githubusercontent.com/95667791/163505650-fe08f2a7-8419-4a2f-97b7-e5182dd386e1.png" alt="Discord" data-size="line"> Discord Claimens#3460<br><img src="https://user-images.githubusercontent.com/95667791/163506639-7d27e947-94c7-4740-bb46-8e5791e9fc6c.png" alt="Telegram" data-size="line"> Telegram @clemensg</p>                                                         | <p>🔒 Delegate to CryptoCrew Validators on: <a href="https://wallet.keplr.app/#/osmosis/stake?modal=detail&#x26;chainId=osmosis-1&#x26;validator=osmovaloper1h2c47vd943scjlfum6yc5frvu2l279lwjep5d6">Osmosis</a>, Cosmos Hub, Terra, Juno, Band Protocol, Stargaze, OmniFlix, Evmos, Gravity Bridge, Comdex, Chihuahua, e-Money, Lum, Bitsong, Vidulum, Dig, Odin<br>🙏 Donations accepted @ <code>osmo15md2qvgma8lnvqv67w0umu2paqkqkheg332u7d</code></p> |
+| [Cosmonaut Stakes](https://twitter.com/CosmonautStakes)   | <p><img src="https://user-images.githubusercontent.com/95667791/163506878-e7e278b2-1cf6-4dac-bc13-e92f63600ee0.png" alt="Twitter" data-size="line"> **Twitter:** <a href="https://twitter.com/CosmonautStakes">@CosmonautStakes</a><br><img src="https://user-images.githubusercontent.com/95667791/163505650-fe08f2a7-8419-4a2f-97b7-e5182dd386e1.png" alt="Discord" data-size="line"> **Discord:** Danoly \| Cosmonaut Stakes#6003<br><img src="https://user-images.githubusercontent.com/95667791/163506639-7d27e947-94c7-4740-bb46-8e5791e9fc6c.png" alt="Telegram" data-size="line"> **Telegram:** <a href="https://t.me/CosmonautStakes">@CosmonautStakes</a></p>                             | <p>🔒 Delegate to **Cosmonaut Stakes** on: <a href="https://wallet.keplr.app/chains/persistence?modal=validator&chain=core-1&validator_address=persistencevaloper1rzauu3undh97yvdnj7wu2wwstm9wj8heeq2vcz&referral=true">Persistence</a></p>                                                                                                                                                                                                               |
```

Comment:
The image is very big in preview. Can add `width` b/w 20-30 in the `img` tag (whichever looks fine).
Also if you can add width for all the rows, it'd be great.

Reply:
I've updated the width of all images and added an emoji for the email to follow the same pattern as the other social media links. I hope this is what you were referring to when saying "**add width for all the rows**".

Reply:
Great, Thank 👍